druid-shell/src/backend/x11/application.rs

```diff
@@ -778,14 +778,18 @@ impl Application {
 
         // from gettext manual
         // https://www.gnu.org/software/gettext/manual/html_node/Locale-Environment-Variables.html#Locale-Environment-Variables
-        var_non_empty("LANGUAGE")
+        let mut locale = var_non_empty("LANGUAGE")
```

Comment:
Would it be possible to make var_non_empty log some info (ideally at some verbose level if such a thing is available). It can be useful to the user to know which var is used and which value it has. Currently if the variable cannot be parse you just see "en_US".

Reply:
I will leave this open for whoever is doing the final review. It does hinder readability but I can see its usefulness.

Reply:
I think this would be welcome, but I'd make it a separate PR.